lib/active_merchant/billing/gateways/world_net_v2/gateway.rb

```diff
@@ -0,0 +1,300 @@
+require 'active_merchant/billing/gateways/world_net_v2/token'
+require 'ipaddr'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    module WorldNetV2 #:nodoc:
+      class Gateway < Gateway
+        UnsupportedActionError = Class.new(StandardError)
+        Invalid3DSVersionError = Class.new(StandardError)
+
+        self.test_url = 'https://testpayments.worldnettps.com/merchant/api/v1'
+        self.live_url = 'https://example.com/live'
+
+        self.supported_countries = ['US']
+        self.default_currency = 'USD'
+        self.supported_cardtypes = [:visa, :master, :american_express, :discover]
```

Comment:
@mark should weigh in on supported card types.

Reply:
Oops, I think @marknfischer